wp-admin/js/widgets/media-widgets.js

```diff
@@ -429,21 +428,26 @@ wp.mediaWidgets = ( function( $ ) {
 		 * @returns {void}
 		 */
 		selectMedia: function selectMedia() {
-			var control = this, selection, mediaFrame, defaultSync;
+			var control = this, selection, mediaFrame, defaultSync, mediaFrameProps;
 
 			if ( control.isSelected() && 0 !== control.model.get( 'attachment_id' ) ) {
 				selection = new wp.media.model.Selection( [ control.selectedAttachment ] );
 			} else {
 				selection = null;
 			}
 
+			mediaFrameProps = control.mapModelToMediaFrameProps( control.model.toJSON() );
+			control.displaySettings.set( 'size', mediaFrameProps.size );
```

Comment:
Good question. It is image specific, but the underlying display settings model seems to populate regardless: https://github.com/WordPress/wordpress-develop/blob/4.7.3/src/wp-includes/js/media-views.js#L1201

Even when it's not an image: https://github.com/WordPress/wordpress-develop/blob/4.7.3/src/wp-includes/js/media-views.js#L1231-L1241

But humm, the template does seem to check for existence: https://github.com/WordPress/wordpress-develop/blob/4.7.3/src/wp-includes/media-template.php#L680-L708

So yeah, better safe than sorry and do an explicit check here. 👍